webapp/src/main/webapp/WEB-INF/conf/news/news-configuration.xml

```diff
@@ -95,6 +95,14 @@
           <description>The resources that start with the following package name should be load from file system</description>
           <value>locale.portlet.news.News</value>
         </values-param>
+        <values-param>
+          <name>init.resources</name>
+          <value>locale.navigation.group.platform.web-contributors</value>
+        </values-param>
+        <values-param>
+          <name>portal.resource.names</name>
+          <value>locale.navigation.group.platform.web-contributors</value>
```

Comment:
- to be added into existing above values param portal.resource.names
- <description> to be removed from classpath.resources and portal.resource.names values params